src/fetch/payouts.py

```diff
@@ -0,0 +1,340 @@
+"""Logic for Post CIP 20 Solver Payout Calculation"""
+from __future__ import annotations
+
+import logging
+import math
+from dataclasses import dataclass
+from datetime import timedelta
+from typing import Callable
+
+import pandas
+from dune_client.types import Address
+from pandas import DataFrame, Series
+
+from src.constants import COW_TOKEN_ADDRESS
+from src.fetch.dune import DuneFetcher
+from src.fetch.prices import eth_in_token, TokenId, token_in_eth
+from src.models.accounting_period import AccountingPeriod
+from src.models.overdraft import Overdraft
+from src.models.token import Token
+from src.models.transfer import Transfer
+from src.pg_client import MultiInstanceDBFetcher
+
+# TODO - this should be a runtime parameter.
+PERIOD_BUDGET_COW = 1
```

Comment:
I am sure that this value is not going to be equal to 1 WEI. cc @fhenneke and @harisang

Reply:
Using the number 383307 here (according to @fhenneke) this is the budget.

Reply:
the total weekly budget is around 383307 COW tokens.
more precisely it is 20_000_000 / 365.2425 * 7 [yearly budget divided by days in a year multiplied by days in a week]